File: starters/rmsbolt.rs
// rust rmsbolt starter file

// Local Variables:
// rmsbolt-command: "rustc -C opt-level=0"
// rmsbolt-disassemble: nil
// End:


fn is_rms(a: char) -> i32 {
    match a {
        'R' => 1,
        'M' => 2,
        'S' => 3,
        _ => 0,
    }
}

fn main() {
    let a: u8 = 1 + 1;
    if is_rms(a as char) != 0 {
        println!("{}", a);
    };
}

File: starters/rmsbolt.go
package main

import (
	"fmt"
)

// Go rmsbolt starter file

// Local Variables:
// rmsbolt-command: "go"
// rmsbolt-disassemble: t
// End:

func isRMS(a int) int {
	switch a {
	case 'R':
		return 1
	case 'M':
		return 2
	case 'S':
		return 3
	default:
		return 0
	}
}

func main() {
	a := 1 + 1
	if isRMS(a) != 0 {
		fmt.Printf("%c\n", a)
	}
}
